lib/OpenQA/Worker/Job.pm

```diff
@@ -429,7 +429,7 @@ sub _stop_step_4_upload {
                 my @other = (
                     @{find_video_files($pooldir)->map('basename')->to_array},
                     (COMMON_RESULT_FILES,
-                        qw(serial0 video_time.vtt serial_terminal.txt virtio_console.log virtio_console1.log)
+                        qw(serial0 video_time.vtt serial_terminal.txt virtio_console.log virtio_console1.log worker_packages.txt)
```

Comment:
Since you're adjusting both occurrences of `COMMON_RESULT_FILES` it would make sense to add the file to that constant.